### DISCUSSION TITLE
新赚吧 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#75] [模板讨论]
新赚吧 模板问题反馈评论区
有问题可以下面反馈。

**Comment by OwnerCM** [comment]
签到已失效

**Reply by wjf0214** [reply]
已更新。

**Comment by stkroot** [comment]
会不断报错失败，然后超过限制次数自动禁用，但实际签到成功

Failed at 5/8 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 302: Found,
Request URL: https://v1.xianbao.net/app.php?p=sign&action=todaysign

**Reply by wjf0214** [reply]
已修复。

**Reply by stkroot** [reply]
还是不行

**Comment by stkroot** [comment]
还是不行， https://v1.xianbao.net/app.php?p=sign&action=todaysign 现在会跳转到这个连接  https://v1.xianbao.net/app.php?p=robot&action=sign

**Reply by wjf0214** [reply]
是啊，我前两周更新的可以的。模板里就有跳转的这个链接。

**Reply by stkroot** [reply]
要不然把签到结果验证去掉？旧版签到，实际是成功的

只是因为返回报错，然后循环错误几次后，脚本就自动停止了
